src/Network.java

```diff
@@ -123,10 +126,12 @@ public void run() {
 
                 // For each example we set the input layer's node's inputs to the example value,
                 // then calculate the output for that example.
-                examples.forEach(example -> {
+                examples.stream().forEach(example -> {
                     Double networkOutput = forwardPropagate(example);
                     output.add(networkOutput);
 
+                    //System.out.println(networkOutput);
```

Comment:
Will this be used later?  Commented out code isn't super great.  Maybe we should have a debug mode and levels of debug but that seems a bit much for our time restriction.  If you will use this for debugging later it can stay.